### ISSUE TITLE
Error fetching flag settings: 400

### ISSUE THREAD

**Issue by ahmednkhan24** [launchdarkly/hello-node-client#6]
Hey,

I have an existing React app deployed to production that successfully uses the LaunchDarkly React SDK.

I have a use-case where I need read-only access to the same flags in a client side node application.

I cloned this repo and updated the `environmentId` to the same client id I use for the react app, however I get this error when I run `node index.js`

```
node index.js                                                                                                                                                                                    
error: [LaunchDarkly] Error fetching flag settings: 400
*** SDK failed to initialize: LaunchDarklyFlagFetchError: Error fetching flag settings: 400
```

Any thoughts on what could be going wrong and how to fix? Any tips would be really appreciated.

**Comment by kinyoklion** [comment]
Hello @ahmednkhan24,

What version of node are you using?

Thanks,
Ryan

**Comment by ahmednkhan24** [comment]
Thanks for the quick response @kinyoklion!

Node v14.19.3
NPM 6.14.17
OS Windows 10

**Comment by kinyoklion** [comment]
Hey @ahmednkhan24,

Ok. I have tried node 14.19.3, with NPM 6.14.17, on windows 10. I am not getting any similar behavior. 

Basically the only changes that you have versus this repository is the client side id?

Thanks,
Ryan

**Comment by ahmednkhan24** [comment]
Hey @kinyoklion, thanks for the info! I worked with a senior engineer on our team and we figured out what the problem was.
Our React app uses a `hash` property to prevent users from accessing flags they shouldn't.

We were able to get this repo working with our client id's by adding the hash to the `options` parameter:

```javascript
const ldClient = LaunchDarkly.initialize(environmentId, context, { hash: "myHash123" });
```

Thanks again for the help!
